components/gpio_control/function_calls.py

```diff
@@ -1,11 +1,12 @@
 import logging
 import sys
 from subprocess import Popen as function_call
+import os
+import pathlib
 
 logger = logging.getLogger(__name__)
 
-playout_control = "../../scripts/playout_controls.sh"
-
+playout_control = os.path.abspath(os.path.join(pathlib.Path(__file__).parent.absolute(), "../../scripts/playout_controls.sh"))
```

Comment:
Does the regular gpio control installation still work with this change?
@veloxidSchweiz ?

Reply:
What it does: make the relative path to an absolute one because with USB encoder usage, the working directory is another one, so the file needs to be resolved correctly. Should not affect the GPIO usage, but I can't test it because I don't use GPIO.